clients/web/src/stylesheets/layout/global.styl

```diff
@@ -111,6 +111,20 @@ textarea
 /* Stuff that applies across the top level object lists */
 .g-top-level-list-title-container
   display inline-block
+  overflow hidden
+  white-space nowrap
+  width calc(100% - 350px)
```

Comment:
I know this out of the scope of this PR, but is using this type of rule "mobile" friendly ?